lib/checks/keyboard/frame-focusable-after.js

```diff
@@ -0,0 +1,21 @@
+function frameFocusableAfter(results) {
+  const resultMap = new Map();
+  results.forEach(result => {
+    const element = result.node._element;
+    const res = resultMap.get(element) || true;
+    resultMap.set(element, res && result.result);
+  });
+
+  const finalResults = [];
+  const entries = resultMap.entries();
+  for (const entry of entries) {
+    const [element, res] = entry;
```

Comment:
Don't think we need to declare those other variables.

```suggestion
  for (const [element, res] of resultMap.entries()) {
```